gnomad/utils/constraint.py

```diff
@@ -717,9 +725,14 @@ def build_models(
 
         # Generate a Table with all necessary annotations (x and y listed above)
         # for the coverage model.
-        low_cov_group_ht = low_cov_ht.group_by(
-            log_coverage=hl.log10(low_cov_ht.exome_coverage)
-        ).aggregate(
+        if cov_model_type == "logrithmic":
+            cov_value = hl.log10(low_cov_ht[coverage_metric])
+        elif cov_model_type == "linear":
+            cov_value = low_cov_ht[coverage_metric]
+        else:
+            raise ValueError("cov_model_type must be one of 'logrithmic' or 'linear'!")
```

Comment:
Maybe move the error raise to the top of the function, or actually, can we just make this parameter a bool?

Reply:
can't the boolean default just change?